contact_edit.php

```diff
@@ -0,0 +1,25 @@
+<?php
+include "connection.php";

```

Comment:
Should use require instead of include.... if include file fails.. the script will continue which is not what you want.